recipe/meta.yaml

```diff
@@ -17,10 +17,10 @@ build:
 
 requirements:
   host:
-    # Cannot use flit-core >3.3.0 due to cyclic dependency
+    # Cannot use flit-core >3.3.0 due to cyclic dependency 
     - flit-core 3.3.0
     - pip
-    - python >=3.6
+    - python
   run:
     - python >=3.6
```

Comment:
I think a skip section for py<36 makes sense here

Reply:
Done